.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -25,42 +26,55 @@ export const GlobalHeader: React.FC<Props> = ({ className, ...other }) => {
 
   const toggleMenu = () => {
     setisActive(!isActive);
+    if (isActive) {
+      document.body.classList.remove('eds-is-disabled');
+    } else {
+      document.body.classList.add('eds-is-disabled');
+    }
   };
 
   const componentClassName = clsx(styles['global-header'], className, {
     [styles['is-active']]: isActive,
   });
   return (
     <Header behavior="sticky" className={componentClassName} {...other}>
-      <div className={styles['global-header__header']}>
-        <Logo href="#" />
+      <Logo className={styles['global-header__logo']} href="#" />
 
-        <Button
-          className={styles['global-header__menu-button']}
-          variant="bare"
-          text={isActive ? 'Close' : 'Menu'}
-          iconPosition="before"
-          iconName={isActive ? 'x' : 'menu'}
-          inverted={true}
-          onClick={toggleMenu}
-        />
-      </div>
+      <Button
+        className={styles['global-header__menu-button']}
+        variant="bare"
+        aria-label={isActive ? 'Close' : 'Menu'}
```

Comment:
@anniehu4 Ah, I _knew_ there was a reason why we didn't use `aria-label` , but found the article: https://heydonworks.com/article/aria-label-is-a-xenophobe/

I'm not sure how true that post is (it's from 2018, so things could have improved), but at least now you know why we had things set up the way we had them.

Reply:
Oh interesting! I hadn't heard about that. Thanks for the link!